pkg/github/bumper/bumper.go

```diff
@@ -261,8 +229,8 @@ func Run(_ context.Context, o *Options, prh PRHandler) error {
 }
 
 func processGitHub(o *Options, prh PRHandler) error {
-	stdout := CensoredWriter{Delegate: os.Stdout, Censor: secret.Censor}
-	stderr := CensoredWriter{Delegate: os.Stderr, Censor: secret.Censor}
+	// stdout := CensoredWriter{Delegate: os.Stdout, Censor: secret.Censor}
+	// stderr := CensoredWriter{Delegate: os.Stderr, Censor: secret.Censor}
```

Comment:
remove if not needed

Reply:
Yeah, it's not needed, I missed it during clean up.